rules/performance/tests/Rector/FuncCall/PreslashSimpleFunctionRector/PreslashSimpleFunctionRectorTest.php

```diff
@@ -16,6 +17,12 @@ final class PreslashSimpleFunctionRectorTest extends AbstractRectorTestCase
      */
     public function test(SmartFileInfo $fileInfo): void
     {
+        $filename = sprintf('%s.php.inc', Option::AUTO_IMPORT_NAMES);
+
+        if ($fileInfo->endsWith($filename)) {
+            $this->setParameter(Option::AUTO_IMPORT_NAMES, true);
+        }
+
         $this->doTestFileInfo($fileInfo);
```

Comment:
This should be handled in a stanalone new test case.

Reply:
implemented.